src/app/index.controller.js

```diff
@@ -55,6 +55,9 @@
             }
         };
 
+        $rootScope.hasAdminRole = function() {
```

Comment:
Nit (just something I noticed, but I don't know your application well): would it make sense to have a `hasRole` function where you pass in the role as an argument and then get a `bool` back, which would allow you to use it for other roles (e.g. chairperson)?

Reply:
The usage in this case is for a simple role-based check in the html to control content display, so we definitely need something that is parameterless and returns a boolean since the html won't know what the roles are. I can imagine the method more generalized here (in `index.controller.js`) and have callers in the individual page controllers be more specific.